lib/ts/recipe/emailverification/recipe.ts

```diff
@@ -209,4 +217,105 @@ export default class Recipe extends RecipeModule {
     addGetEmailForUserIdFunc = (func: GetEmailForUserIdFunc): void => {
         this.getEmailForUserIdFuncsFromOtherRecipes.push(func);
```

Comment:
where and how can we add a resync for the account linking claim? For example, if the if it's left in the session and the table doesn't have it, we need to update the session at some point independently to the email verification API being completed.